### DISCUSSION TITLE
Can't Submit Application for Student Dev Pack

### DISCUSSION THREAD

**Discussion by ansuhayda** [community/community#121264] [GitHub Education]
### Hi everyone,

I'm beyond frustrated.  I've been trying to submit my student info for an entire afternoon but keep failing.  I was finally able to get my student ID uploaded but it seems there's still an issue. I see a message that "the item you uploaded is insufficient to demonstrate your current academic status." I uploaded a picture of my student ID so I don't know why that would be insufficient. There is a suggestion telling me, "Attendees of this school must verify and use their academic email address" which I have already verified my school email address within my Github profile. The next suggestion is to, "Please use your device camera to submit academic affiliation documents."  Well, I've been unable to upload pictures of any other documents despite trying many times.  I've tried uploading pictures of my transcript, my course enrollment, and an invoice for classes.  Nothing works.  I keep getting kicked back to the page to submit documents.  I'm pretty irritated at this point and don't know what to do next so any help would be greatly appreciated!

**Comment by akshatjaiin** [comment]
I m facing the same issue when I m leaving the email field empty 😁 i submitted with a random . In the school email field waiting to get rejected again

**Comment by samus-aran** [comment]
Hi @ansuhayda 

We've updated our [Student Developer Pack Application FAQs & Common Rejection Reasons](https://github.com/orgs/community/discussions/111352) discussion. It's packed with valuable information that might just have what you're looking for. 

Here is also a few tips:

- It would be preferable that a student is on campus when applying and thus able to upload a document but with remote learning that isn't always possible, **so we request students to submit a live photo of their evidence rather than an upload.**
- Use an academic email. Some schools may require students to be verified by an already confirmed domain distributed by the school.
- Ensure your name, and school name is on the evidence. If you don't have a single piece that contains all of these then take a live capture of multiple evidence in a single shot.

**Comment by AdrianeNS** [comment]
I was finally able to get it to process my student ID but I received an error that my ID does not have the name of my college on it, however, the name of my college is clearly written across the bottom of the card.  I also received the same error when I was finally able to get it to upload my other documents including my transcript and my course enrollment for the semester.  Does anyone from Github even read these forums?  I see post after post of those experiencing similar issues with no replies from Github.

**Comment by Im-Ayetskie** [comment]
Is the file under 1mb? I think the file size should be 1mb or less for them to take the picture.

**Comment by akshatjaiin** [comment]
its been months i m facing the same issue
